### PR TITLE
Fix Whitespace in Slides

### PR DESCRIPTION
**Pull request by pts-sergiomartins**
Slides 48 - 53 had indentation issues that prevented copy-pasting into the python console.

Also removed trailing whitespace on lines in the English and French HTML files.

### CODE REVIEW

File: slides-fr.html [hunk 1]
```diff
@@ -33,7 +33,7 @@
   <header class="slide">
     <img class="logo" src="framework/img/logo-llc-white.png" alt="Ladies Learning Code logo">
     <h1>Bienvenue chez Ladies Learning Code!</h1>
-      
+
```
Comment (pts-sergiomartins) [inline slides-fr.html]: My editor trims trailing whitespace when I hit save, but this is not the focus of this Pull Request. Keep scrolling down :)

File: slides.html [hunk 2]
```diff
@@ -1172,21 +1172,21 @@ <h1 class="heading-bg"><span>And now for something completely different</span></
         1. pass the file into `DictReader`
 
         ```
-         import csv
-         with open('llc-chapters.csv') as chapters_file:
-             chapters_data = csv.DictReader(chapters_file)
+        import csv
+        with open('llc-chapters.csv') as chapters_file:
+            chapters_data = csv.DictReader(chapters_file)
```
Comment (pts-sergiomartins) [inline slides.html]: Fixed intendation so copy-pasting into the console works

File: slides.html [hunk 3]
```diff
@@ -1198,27 +1198,27 @@ <h1 class="heading-bg"><span>And now for something completely different</span></
...
         Python can 'look up' each value in a row just as you would the meaning of the word *python*.
-        
+
         ```python
-         print(chapter['City'])
-         print(chapter['Province'])
-         print(chapter['Chapter Lead(s)'])
+        print(chapter['City'])
+        print(chapter['Province'])
+        print(chapter['Chapter Lead(s)'])
```
Comment (pts-sergiomartins) [inline slides.html]: Fixed here as well

File: slides.html [hunk 4]
```diff
@@ -1228,20 +1228,19 @@ <h1 class="heading-bg"><span>And now for something completely different</span></
...
-             for chapter in chapters:
-                       print(chapter['City'] + ', ' + chapter['Province'])
-                       print(chapter['Chapter Lead(s)'])
-
+        import csv
+        with open('llc-chapters.csv') as chapters_file:
+            chapters = csv.DictReader(chapters_file)
+            for chapter in chapters:
+               print(chapter['City'] + ', ' + chapter['Province'])
+               print(chapter['Chapter Lead(s)'])
```
Comment (pts-sergiomartins) [inline slides.html]: Indentation with 8+ spaces seems to push content to a new line in Firefox. Fixed that here

File: slides.html [hunk 5]
```diff
@@ -1286,12 +1285,11 @@ <h1 class="heading-bg"><span>And now for something completely different</span></
 
         with open('llc-chapters.csv') as csv_file:
             csv_data = csv.DictReader(csv_file)
-
             count = 0
             for row in csv_data:
                count = count + 1
 
-            print("There are " + str(count) + " chapters")
+        print("There are " + str(count) + " chapters")
```
Comment (pts-sergiomartins) [inline slides.html]: Fixed this indentation issue causing a bug here

**Comment by dmilliken** [comment]
We have been making updates to this workshop, I will take these under consideration. Thanks!

**Pull request closed without merging**